src/components/collapsible-section/collapsible-section.tsx

```diff
@@ -83,10 +84,16 @@ export class CollapsibleSection {
     }
 
     public render() {
+        const bodyId = createRandomString();
```

Comment:
Generating a new body id on every render seems problematic. Better to do it only once when the component is created, for example:

```ts
export class CollapsibleSection {
    // ...
    private bodyId = createRandomString();
    // ...
}    
```

Reply:
thanks, added a fixup!